packages/framework/src/parser.ts

```diff
@@ -137,7 +138,7 @@ export const EnsembleParser = {
         menu ||
         (screens as EnsembleScreenModel[]).find((screen) => screen.isRoot) ||
         screens[0],
-      themes,
+      themes: themes || { default: defaultThemeDefinition },
```

Comment:
Shouldn't this be an array?

Reply:
No, this will be object, so we can select any selected theme from themes using theme name